### DISCUSSION TITLE
my rog strix g18 2023 rgb

### DISCUSSION THREAD

**Discussion by PoepieFloepie** [seerge/g-helper#1982] [General]
i have the ROG Strix G18 (2023) G814 G814JI-N6079W

this device has only a front rgb lightbar and keyboard and no lid or logo rgb
and in the app i see all four
please check...

i also dont see a option to undervolt ,i can set it in the bios to on but must use intel app to undervolt ,so that would be nice to have and max to undervolt to -80 what asus also do with this version 2023

**Comment by seerge** [comment]
@PoepieFloepie if your device has only 4-zone rgb (i.e. no per-key RGB), i can add it to the list of ones with limited RGB support :) then checkboxes would disappear as well

As for undervolting - app offers it only for AMD devices. For Intel you can use BIOS option, or Intel own https://www.intel.com/content/www/us/en/download/17881/intel-extreme-tuning-utility-intel-xtu.html

**Comment by PoepieFloepie** [comment]
what do you mean per key rgb

i can setup every key to a different color with aura creator
my device only support for rgb for keys and front bar ,thats the only rgb

please not all checkboxes gone then leave them as they are

**Reply by seerge** [reply]
@PoepieFloepie ok, then it's a very unusual combination. Most of the devices that have only lightbar also don't have per-key RGB, so currently this things are bundled together :)

I assume having extra checkboxes don't harm
